.zenodo.json

```diff
@@ -88,6 +88,10 @@
       "affiliation": "Department of Psychology, Stanford University",
       "orcid": "0000-0003-3321-7583"
     }
+    {
+      "name": "Romain, Valabregue",
+      "affiliation": "CENIR, ICM, Paris"
+    }
```

Comment:
Please move this above Russ' entry (it and Chris' should be last). And you may want to consider registering for an [ORCID](https://orcid.org/) (or adding yours, if you already have one). It helps link your research contributions across a variety of media.